### ISSUE TITLE
ERROR WHILE USING GUI

### ISSUE THREAD

**Issue by gokuLc2127** [epsylon/ufonet#275]
(exo-open:102121): dbind-WARNING **: 13:37:39.316: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-38IVGD4DAl: Connection refused

(exo-helper-2:102123): dbind-WARNING **: 13:37:39.336: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-38IVGD4DAl: Connection refused
Running Firefox as root in a regular user's session is not supported.  ($XAUTHORITY is /home/gokul/.Xauthority which is owned by gokul.)

![dddosgui](https://user-images.githubusercontent.com/71785777/94360032-25a59580-00c8-11eb-8b4c-926a60a3e0d6.png)

**Comment by epsylon** [comment]
@gokuLc2127 this is because Firefox doesn't allow you to open a session using root. You can try to **execute UFONet without using root** (the tool will ask you for that permissions when required).
Or, you can try to **open manually** the gui (using root) opening on your browser this url: **http://127.0.0.1:9999**

**Comment by gokuLc2127** [comment]
Thank u bro
